renderer-macos/lib/Styles.re

```diff
@@ -31,6 +36,28 @@ let setViewStyle = (view: view, attribute: [> viewStyle]) =>
       let {r, g, b, a}: Color.t = color;
       BriskView.setBorderColor(view, r, g, b, a);
     };
+  | `Shadow(({x, y, opacity, blur, color}: Shadow.t)) =>
+    let x = isUndefined(x) ? 0. : x;
+    let y = isUndefined(y) ? 0. : y;
+
+    BriskView.setShadowOffset(view, x, y);
+
+    if (!isUndefined(opacity)) {
+      BriskView.setShadowOpacity(view, opacity);
+    };
+    if (!isUndefined(blur)) {
+      BriskView.setShadowRadius(view, blur);
+    };
+
+    if (color !== Color.undefined) {
+      let {r, g, b, a}: Color.t = color;
```

Comment:
Same trick can be used here: `let {Color.r, g, b, a} = color;`

Reply:
This one in particular didn't work for me, I guess because I was shadowing the original module from `color` package.